tools/clang/tools/dxcompiler/CMakeLists.txt

```diff
@@ -128,7 +128,7 @@ if (WIN32)
 endif()
 target_link_libraries(dxcompiler PRIVATE ${LIBRARIES})
 if (ENABLE_SPIRV_CODEGEN)
-  target_link_libraries(dxcompiler PRIVATE clangSPIRV)
+  target_link_libraries(dxcompiler PRIVATE clangSPIRV spirv_to_dxil)
```

Comment:
I think rather than linking this, it'd make sense to use `LoadLibrary()`/`GetProcAddress()`. That way you don't even need the .lib, and just the DLL at runtime - and only if it's actually being used.